components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/util/designview/designgenerator/generators/TriggerConfigGenerator.java

```diff
@@ -48,11 +49,14 @@ public TriggerConfigGenerator(String siddhiAppString, Map<String, StreamDefiniti
      */
     public TriggerConfig generateTriggerConfig(TriggerDefinition triggerDefinition) throws DesignGenerationException {
         String at = "";
```

Comment:
shall we change the parameter name "at" for a descriptive name please

Reply:
Fixed in eceb77b51dbb904e257a9b9b5f72d8e14560d910